pekko-http-cors/src/main/scala-2.13/ch/megard/pekko/http/cors/OptionConverters.scala

```diff
@@ -0,0 +1,8 @@
+package ch.megard.pekko.http.cors
+
+import java.util.Optional
+
+object OptionConverters {
+  @inline final def toScala[A](o: Optional[A]): Option[A] = scala.jdk.javaapi.OptionConverters.toScala(o)
+  @inline final def toJava[A](o: Option[A]): Optional[A]  = scala.jdk.javaapi.OptionConverters.toJava(o)
```

Comment:
Does this not live in Pekko?

Reply:
Its package private and also marked as `@InternalApi`

Reply:
I see, the package here is `ch.megard.pekko.http.cors`